src/command_modules/azure-cli-container/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+0.1.22
+++++++
+* Add Git Repo volume mount parameters '--gitrepo-repository' '--gitrepo-directory' and '--gitrepo-mount-path'
```

Comment:
The names are kinda long no?

Reply:
How about `--gitrepo-url` `--gitrepo-dir`? I also want to add `--gitrepo-revision` in another iteration.

Reply:
@yolo3301 yes like those better.